torch/serialization.py

```diff
@@ -338,7 +338,10 @@ def save(obj, f, pickle_module=pickle, pickle_protocol=DEFAULT_PROTOCOL, _use_ne
            containing a file name
         pickle_module: module used for pickling metadata and objects
         pickle_protocol: can be specified to override the default protocol
-
+    
```

Comment:
The Linter is complaining that lines 341 and lines 344 contains whitespace: https://github.com/pytorch/pytorch/pull/39303/checks?check_run_id=739499514

Reply:
Resolved the issue and rebased the repo as well.